src/main/index.ts

```diff
@@ -61,11 +61,15 @@ import { FilesystemProvisionerStore } from "./extension-filesystem";
 import { SentryInit } from "../common/sentry";
 import { ensureDir } from "fs-extra";
 import { Router } from "./router";
-import { initMenu } from "./menu";
+import { initMenu } from "./menu/menu";
```

Comment:
why change this import?

Reply:
Moved file to separate directory. Prefer not to create `index.ts` when really not needed.

Reply:
Okay I guess, I find `index` files much cleaner to read especially since the rest of this repo we use them.

Reply:
We have seen both used in this repo. We prefer it this way so that we don’t need so much boilerplate code. 

Not that big deal though for us

Reply:
To me it seems that more common practice is to use "index" in this repo but that has a downside because it requires tree-shaking (which might not work in practice too well). Without tree-shaking we are actually "loading" all dependencies from index ... of course import shouldn't have side-effects in a perfect world (we are not there yet).

Reply:
Hmm. The index-files do feel like an unnecessary indirection and complication. We could consider if the benefits of encapsulation really overweigh them. As said, not a bid deal, as there's far bigger fish out there :)

...but in general, when it comes to side-effects on import, I'm afraid it's not something our team should tolerate at all. Let's make a hard line for this, right guys? :)

Reply:
I agree, it also makes circular dependencies too easy